crates/turbopack-node/src/render/render_static.rs

```diff
@@ -91,7 +91,11 @@ async fn static_error(
     operation: Option<NodeJsOperation>,
     fallback_page: DevHtmlAssetVc,
 ) -> Result<AssetContentVc> {
-    let message = format!("{error:?}");
+    let message = format!("{error:?}")
+        // TODO this is pretty inefficient
+        .replace('&', "&amp;")
+        .replace('>', "&gt;")
+        .replace('<', "&lt;");
```

Comment:
Could use https://docs.rs/html-escape/latest/html_escape/fn.encode_text.html instead.

Reply:
Good idea. I added this as issue